src/Umbraco.Core/Models/ImageUrlGenerationOptions.cs

```diff
@@ -1,66 +1,68 @@
-﻿namespace Umbraco.Cms.Core.Models
+namespace Umbraco.Cms.Core.Models
 {
     /// <summary>
-    /// These are options that are passed to the IImageUrlGenerator implementation to determine
-    /// the propery URL that is needed
+    /// These are options that are passed to the IImageUrlGenerator implementation to determine the URL that is generated.
     /// </summary>
     public class ImageUrlGenerationOptions
     {
-        public ImageUrlGenerationOptions (string imageUrl)
-        {
-            ImageUrl = imageUrl;
-        }
+        public ImageUrlGenerationOptions(string imageUrl) => ImageUrl = imageUrl;
 
         public string ImageUrl { get; }
+
         public int? Width { get; set; }
+
         public int? Height { get; set; }
-        public decimal? WidthRatio { get; set; }
-        public decimal? HeightRatio { get; set; }
+
         public int? Quality { get; set; }
+
         public ImageCropMode? ImageCropMode { get; set; }
+
         public ImageCropAnchor? ImageCropAnchor { get; set; }
-        public bool DefaultCrop { get; set; }
+
         public FocalPointPosition FocalPoint { get; set; }
+
         public CropCoordinates Crop { get; set; }
+
         public string CacheBusterValue { get; set; }
+
         public string FurtherOptions { get; set; }
-        public bool UpScale { get; set; } = true;
-        public string AnimationProcessMode { get; set; }
 
         /// <summary>
-        /// The focal point position, in whatever units the registered IImageUrlGenerator uses,
-        /// typically a percentage of the total image from 0.0 to 1.0.
+        /// The focal point position, in whatever units the registered IImageUrlGenerator uses, typically a percentage of the total image from 0.0 to 1.0.
         /// </summary>
         public class FocalPointPosition
         {
-            public FocalPointPosition (decimal top, decimal left)
+            public FocalPointPosition(decimal left, decimal top)
```

Comment:
I've changed the parameter order of this constructor to align it with the way it's used by ImageSharp and the property order below (including the order in which they are set).

We might also want to change the names to `x` and `y`. And while we're doing that, change from `decimal` to `float` (as we don't need the additional precision) and convert this class into a struct, as that has additional performance improvements: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct.

Reply:
I think this is probably OK named as is.  `x` would correspond to `left`, but `y` arguably makes more sense related to the `bottom` rather than `top`, as typical X/Y coordinates are plotted.  With `left` and `top` we resolve any ambiguiity.

Reply:
It's more to align to the naming used by Microsoft and ImageSharp for [rectangles](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.rectangle.-ctor?view=net-5.0) and [points](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.point?view=net-5.0). Both have aliases for left and top (only property getters), but the constructors always reference X and Y.

Also keep in mind that these aren't coordinates as used in graphs and images use the upper-left corner as origin in basically all editing programs.

Reply:
Maybe it makes more sense to just use the `PointF` and `RectangleF` structs from [`System.Drawing.Primitives`](https://www.nuget.org/packages/System.Drawing.Primitives/)? Lets do that in a new PR though 😉